backend/api/v1/v1_users/tests/tests_user_invitation.py

```diff
@@ -21,12 +21,12 @@ def test_user_list(self):
         users = response.json()
 
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(users[0]['first_name'], 'Admin')
-        self.assertEqual(users[0]['last_name'], 'RTMIS')
-        self.assertEqual(users[0]['email'], 'admin@rtmis.com')
-        self.assertEqual(users[0]['administration'],
+        self.assertEqual(users['data'][0]['first_name'], 'Admin')
+        self.assertEqual(users['data'][0]['last_name'], 'RTMIS')
+        self.assertEqual(users['data'][0]['email'], 'admin@rtmis.com')
+        self.assertEqual(users['data'][0]['administration'],
                          {'id': 1, 'name': 'Indonesia', 'level': 1})
-        self.assertEqual(users[0]['role'],
+        self.assertEqual(users['data'][0]['role'],
                          {'id': 1, 'value': 'Super Admin'})
 
     def test_add_edit_user(self):
```

Comment:
Add more user with the new user seeder call command, check the pagination objects.